package.json

```diff
@@ -11,7 +11,7 @@
   },
   "main": "dist/official-addthis-angularjs.js",
   "scripts": {
-    "test": "gulp test"
+    "test": "./node_modules/.bin/gulp test"
```

Comment:
This is fine.
Just so you know, the packageJson script refers to packages, so the prev + current instances are pointing to the same local gulp.